neuralmonkey/run.py

```diff
@@ -140,7 +140,7 @@ def main() -> None:
         execution_results, output_data = run_on_dataset(
             CONFIG.model.tf_manager, CONFIG.model.runners,
             dataset, CONFIG.model.postprocess, write_out=True,
-            batch_size=runners_batch_size)
+            batch_size=runners_batch_size, log_progress=5)
```

Comment:
chvíli jsem nad tim přemejšlel a 5 minut mi přijde moc. Dal bych default jednu minutu a psal bych to tam v sekundách.